packages/documentation/pages/usage/components/filter.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div>
+		<ComponentInfo v-bind="{ component }" />
+		<KtFilter
+			:columns="columns"
+			:isLoading="isLoading"
+			:value="selectedFilters"
+			@update-value="updateValue"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import { KtFilter, Kotti } from '@3yourmind/kotti-ui'
+import { defineComponent, ref } from '@vue/composition-api'
+
+import ComponentInfo from '~/components/ComponentInfo.vue'
+
+export default defineComponent({
+	name: 'DocumentationPageUsageComponentsFilter',
+	components: {
+		ComponentInfo,
+	},
+	setup() {
+		const columns: Kotti.Filter.Column[] = [
+			{
+				key: 'column-1',
+				label: 'Column 1',
+				filterDefinition: {
```

Comment:
Why does `filterDefinition` need to be a sub-object here? Just seems to add noise to the usage or is there something I’m missing?